clvm/SExp.py

```diff
@@ -16,27 +17,27 @@
 
 CastableType = typing.Union[
     "SExp",
-    "CLVMObject",
+    CLVMObjectLike,
     bytes,
     str,
     int,
     None,
     list,
-    typing.Tuple[typing.Any, typing.Any],
+    typing.Tuple["CastableType", ...],
```

Comment:
```suggestion
    typing.Tuple["CastableType", "CastableType"],
```
The `...` suggests it can take an arbitrary number of parameters. In this case, it's exactly a pair.

Reply:
Yuck, that looks like a pretty straightforward error on my part there.  Thanks.